jbehave-core/src/main/java/org/jbehave/core/model/Story.java

```diff
@@ -59,6 +60,9 @@ public Story(String path, Description description, Meta meta, Narrative narrativ
     public Story(String path, Description description, Meta meta, Narrative narrative, GivenStories givenStories,
             Lifecycle lifecycle, List<Scenario> scenarios) {
         this.path = path;
+        if (path != null) {
+            this.name = new File(path).getName();
+        }
```

Comment:
I see that this check is from RSP but in what cases can it be null?

Reply:
no idea, this change was done to avoid duplicates, the logic is moved as is